packages/atomic/src/components/atomic-search-interface/atomic-search-interface.tsx

```diff
@@ -337,7 +340,7 @@ export class AtomicSearchInterface {
     return this.i18n.use(Backend).init({
       debug: this.logLevel === 'debug',
       lng: this.language,
-      fallbackLng: ['en'],
+      fallbackLng: false,
```

Comment:
If some strings are missing from the language set by the user, are english strings going to be used?

Reply:
Good point! let me check...

Reply:
This indeed should be an fr/en improvement